GenericsHomework/GenericsHomework/GenericsHomework.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace GenericsHomework
```

Comment:
```suggestion
using System;

﻿namespace GenericsHomework
```

Your code doesn't compile because you are missing a using statement. This is needed because of the exception you are throwing further down in the code.